draft-birkholz-rats-corim.md

```diff
@@ -107,4 +153,13 @@ Security Considerations
 
 See Body {{mybody}}.
 
---- back
+New indexes for Software Tag Link Relationship Values needs to be registered as below:
```

Comment:
You also need to add the `trusted`/`untrusted` registration.

Reply:
Not Applicable now!